subsys/bluetooth/audio/cap_internal.h

```diff
@@ -92,6 +93,12 @@ struct cap_broadcast_reception_start {
 	uint8_t num_subgroups;
 	struct bt_bap_bass_subgroup subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
 };
+
+struct cap_broadcast_reception_stop {
+	uint8_t src_id;
+	uint8_t num_subgroups;
+	struct bt_bap_bass_subgroup subgroups[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
```

Comment:
Do we need to store this statically? The subgroups for the remove source could probably just be stack-allocated as needed, right? Or would that require too much stack memory?

Reply:
It could be stack allocated, but we wouldn't gain anything. The `struct cap_broadcast_reception_stop` is used in the `struct bt_cap_commander_proc_param` as part of a union

```
#if defined(CONFIG_BT_BAP_BROADCAST_ASSISTANT)
		struct cap_broadcast_reception_start broadcast_reception_start;
		struct cap_broadcast_reception_stop broadcast_reception_stop;
#endif /* CONFIG_BT_BAP_BROADCAST_ASSISTANT */
```

so there wouldn't be any memory savings